client/src/views/RESTView.vue

```diff
@@ -82,6 +85,9 @@ export default {
   },
   data() {
     return {
+      resultData: null,
+      json_result: true,
```

Comment:
instead of introducing two variables, I suggest creating a variable named `showResultInTable` witch configured as `false` by default, and set it to true with API is `cat` API.